exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,10 @@
 function halve(number) {
   // complete the function here
+  return number/2;
 }
```

Comment:
Sorry this is going to sound really picky but whilst there is nothing wrong with the code here, it's not particularly tidy or readable. A more readable example would be:
`const halvedNumber = number / 2;

`

Reply:
Done 👍 Thank You sir